libs/external-db-firestore/src/firestore_capabilities.ts

```diff
@@ -0,0 +1,21 @@
+import { AdapterOperators } from '@wix-velo/velo-external-db-commons'
+import { CollectionOperation, DataOperation, FieldType } from '@wix-velo/velo-external-db-types'
+
+const { query, count, queryReferenced, aggregate, } = DataOperation
+const { eq, string_begins, gt, gte, lt, lte, include } = AdapterOperators
+const UnsupportedCapabilities = [DataOperation.insertReferences, DataOperation.removeReferences, DataOperation.queryReferenced]
+
+
+export const ReadWriteOperations = Object.values(DataOperation).filter(op => !UnsupportedCapabilities.includes(op))
+export const ReadOnlyOperations = [query, count, queryReferenced, aggregate]
+export const FieldTypes = Object.values(FieldType)
+export const CollectionOperations = Object.values(CollectionOperation)
+export const ColumnsCapabilities = {
+    text: { sortable: true, columnQueryOperators: [eq, string_begins, include, gt, gte, lt, lte] },
+    url: { sortable: true, columnQueryOperators: [eq, string_begins, include, gt, gte, lt, lte] },
+    number: { sortable: true, columnQueryOperators: [eq, gt, gte, lt, lte, include] },
+    boolean: { sortable: true, columnQueryOperators: [eq] },
+    image: { sortable: false, columnQueryOperators: [] },
+    object: { sortable: false, columnQueryOperators: [eq, string_begins, include, gt, gte, lt, lte] },
```

Comment:
Are we sure about the `object` operators?

Reply:
And if we do, it's not implemented